lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -9,24 +9,27 @@ use Path::Class qw();
 class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
     is => 'Command::V2',
     has_input => [
+        model => {
+            is => 'Text',
+            doc => 'A RNASeq model that has ERCC transcripts spiked in.',
+            is_optional => '1,',
+        },
         bam_file => {
             is => 'FilePath',
-            doc => 'An aligned BAM file to a reference genome without the ERCC transcripts spiked in.',
+            doc => 'A path to a BAM file that has ERCC transcripts spiked in.',
+            is_optional => '1,',
         },
         ercc_fasta_file => {
             is => 'FilePath',
             doc => 'The FASTA format sequence for all ERCC transcripts.',
-            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa'],
+            default_value => '/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa',
```

Comment:
An appended reference sequence exists where the ERCC contigs are appended to the human reference.  The contigs would be in this file:
/gscmnt/gc9008/info/model_data/2861523156/build0bfc1bd5fcfc474c9db737a520ae109d/appended_sequences.fa